### PR TITLE
feat(bingAds): add UET ecomm fields

### PR DESCRIPTION
**Pull request by Gauravudia**
## PR Description

Add UET ecomm fields

## Notion ticket

[Ticket link](https://www.notion.so/rudderstacks/Bing-Ads-Ecomm-53d104e3b79448bb98fa2d926a85cf80)

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: src/integrations/BingAds/browser.js
Comment (MoumitaM) [inline src/integrations/BingAds/browser.js:1]: Replace line 25 with 
`(o.q = w[u]), (w[u] = w.UET ? new UET(o) : []);`

Comment (MoumitaM) [inline src/integrations/BingAds/browser.js:1]: Line 49 
Update the logger message to something like this:
`logger.debug('Trying to load BingAds: https://bat.bing.com/bat.js');`

**Review by MoumitaM — CHANGES_REQUESTED** [review]

File: src/integrations/BingAds/browser.js
Comment (MoumitaM) [reply src/integrations/BingAds/browser.js:1]: Addressed in Different PR

Comment (MoumitaM) [reply src/integrations/BingAds/browser.js:1]: Addressed in Different PR

**Review by MoumitaM — COMMENTED** [review]
LGTM

**Review by MoumitaM — APPROVED** [review]

File: src/integrations/BingAds/utils.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+};
+
+const handleProductsArray = (properties) => {
+  const productIds = [];
+  const items = [];
+  let ecommTotalValue = 0;
+  const { products } = properties;
+  if (Array.isArray(products)) {
+    products.forEach((product) => {
+      if (product) {
```
[the remark this answers is not in the document]
Comment (Gauravudia) [reply src/integrations/BingAds/utils.js:41]: we can remove this. As we have a check on product id on line 44.

[the remark this answers is not in the document]
Comment (Gauravudia) [reply src/integrations/BingAds/utils.js]: updated

**Review by ItsSudip — APPROVED** [review]

**Review by sandeepdsvs — APPROVED** [review]

**Pull request merged**